pipeline/src/resources/Annotator.py

```diff
@@ -4,16 +4,17 @@
 
 
 class Annotator(RequestHandler):
-    def process(self, data):
+    def process(self, data, files=None):
         raise NotImplemented('Annotators must implement `process` method')
 
     def initialize(self):
         self.annotation_types = []
         self._annotations = []
 
     def post(self):
-        body = json.loads(self.request.body.decode())
-        self.process(body)
+        cas = json.loads(self.get_argument('cas', default='{}'))
+        data = self.request.files
+        self.process(cas, data)
```

Comment:
`cas` is mapped to formal `data`, while `data` is formal `files` - can we rename here or at line 7 so that we don't have `data` being passed and a `data` formal where the two don't match?